packages/atomic/src/components/common/pager/pager-common.tsx

```diff
@@ -42,6 +44,15 @@ export const PagerCommon: FunctionalComponent<PagerProps> = (props) => {
     focusOnFirstResultAndScrollToTop();
   };
 
+  const nextIcon = props.nextIcon || ArrowRight;
+  const prevIcon = props.prevIcon || ArrowRight;
+  const defaultIconStyle = 'w-5 align-middle';
+  // Rotate default next arrow icon by 180 to obtain previous arrow icon
+  const prevIconStyle =
```

Comment:
We could also just add an `arrow-left` to our svg assets. It's actually a bit weird that we do not already ship it in our assets ;)

Unless I'm mistaken, the existing arrowRight is a copy of what's available here: https://plasma.coveo.com/foundations/Iconography (arrowHeadRight).

Reply:
https://github.com/coveo/ui-kit/blob/KIT-2104/packages/atomic/src/images/arrow-right.svg is slightly different from https://github.com/coveo/plasma/blob/master/packages/style/resources/icons/svg/arrow-right.svg or https://github.com/coveo/plasma/blob/master/packages/style/resources/icons/svg/arrow-right-rounded.svg

Maybe the one in ui-kit is from a different version of plasma ?

Reply:
Actually [ui-kit's arrow-right-rounded.svg](https://github.com/coveo/ui-kit/blob/KIT-2104/packages/atomic/src/images/arrow-right-rounded.svg) is the same as [plasma's arrow-right-rounded.svg](https://github.com/coveo/plasma/blob/master/packages/style/resources/icons/svg/arrow-right-rounded.svg)

So maybe switch pager to use arrow-right-rounded.svg and add arrow-left-rounded.svg ?

Reply:
Yes, we can do that !

Reply:
Done